material_components_ios.bzl

```diff
@@ -21,6 +21,25 @@ def mdc_objc_library(
       copts = copts,
       **kwargs)
 
+def mdc_private_hdrs_objc_library(
+    name = "privateHeaders",
```

Comment:
name is typically a required field on bazel rules so that it's clear what the dependency name will be without needing to open the definition.